views/pages/categoryRadar.ejs

```diff
@@ -36,6 +36,10 @@
         </ol>
     </div>
 
+    <div class="col-md-12">
```

Comment:
Might not be used directly, but do note that that the Flash messages can be displayed if they are set in cookies. There is a scenario that something went wrong with a particular request, the flash messages were created, but the application failed to redirect the user to a new screen where he would originally view the messages. In such scenario the user would see the messages in any screen he visits next (but this module has to be included on that page)